src/main/java/seedu/address/model/person/Email.java

```diff
@@ -31,7 +31,7 @@ private Email() {
     public Email(String email) throws IllegalValueException {
         requireNonNull(email);
         String trimmedEmail = email.trim();
-        if (!isValidEmail(trimmedEmail)) {
+        if (!isValidEmail(trimmedEmail) && !"Unspecified email".equals(trimmedEmail)) {
```

Comment:
This and the rest of the field classes should return the `UNSPECIFIED` static field if the content matches the unspecified placeholder.